.gitlab-ci.yml

```diff
@@ -6,15 +6,15 @@ stages:
 deploy to data.trezor.io:
   stage: deploy
   needs: []
-  before_script: []
   variables:
     GIT_STRATEGY: clone
   only:
     refs:
       - master
   when: manual
+  before_script: []
```

Comment:
Why move this?

Reply:
just my ocd, i like when before script is just before the script section 🙈